docs/dev/annual_updates.rst

```diff
@@ -197,6 +197,12 @@ to the existing coding tables.
 
 Run a Siloed EIA ETL
 --------------------
+.. note::
+    This section should probably be updated to include reference to the new ``tox`` test
+    called ``get_unmapped_ids`` that was implemented for the FERC1 XBRL integration. We
+    may be able to fully skip this step because ``get_unmapped_ids`` runs the ETL with
+    ``--ignore-foreign-key-constraints`` and saves the unmapped IDs.
+
```

Comment:
I left two of these notes in the docs. I would like to update these sections when we actually integrate a new round of xbrl data. bc i think i know  what needs to be done but i would like to actually run through the steps myself before writing it down.